koshigoe/brick_ftp
# 95 add display key to history

## What did you implement:

close #95 
Add display key to history objects

## How did you implement it:

Add display attribute to 
```
lib/brick_ftp/api/history/file.rb
lib/brick_ftp/api/history/folder.rb
lib/brick_ftp/api/history/login.rb
lib/brick_ftp/api/history/site.rb
lib/brick_ftp/api/history/user.rb
```

Add display argument and documentation to
```
lib/brick_ftp/client.rb
```

## How can we verify it:
The easiest way to verify is to run:
```
BrickFTP::Client.new.list_site_history(display: :full)
```
and look for display attribute presence.

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
